src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement, useState } from 'react'
+import { TAvatarProps } from './types'
+import classNames from 'classnames'
+import FileUpload from '../FileUpload'
+import { FileUploadMode } from '../FileUpload/types'
+import '../../assets/styles/components/_avatar.scss'
+
+export const Avatar = ({
+  id,
+  color = 'default',
+  size = 'medium',
+  imagePath = '',
+  initials = '',
+  className = '',
+  isEditable = false,
+  onAvatarChange
+}: TAvatarProps): ReactElement => {
+  const [image, setImage] = useState<string>(imagePath)
```

Comment:
because imagePath type already string you can write just 
const [image, setImage] = useState(imagePath)